app/src/main/java/org/imozerov/streetartview/ui/extensions/MapExtensions.kt

```diff
@@ -56,7 +64,7 @@ fun GoogleMap.addUserLocationMarker(userLocation: LatLng) {
 
 fun GoogleMap.addArtObject(artObject: ArtObjectUi) {
     val markerOptions = MarkerOptions().position(LatLng(artObject.lat, artObject.lng))
-            .title(artObject.authorsNames())
+            .title(artObject.id)
```

Comment:
As is. Marker has no fields to store our id. We need to use marker as view holder here to handle onClick properely